packages/backend/migrations/20250409171833_add_permissions_object_subject_idx.up.sql

```diff
@@ -0,0 +1,5 @@
+-- no-transaction
```

Comment:
FYI, @jmoggr, I had to add this to fix the migrations:

```
~/Local/topos/catcolab/packages/backend> sqlx database reset
Drop database at postgres://epatters:catdev@localhost:5433/catcolab? (y/n): y
Applied 20241004010448/migrate document refs (34.13997ms)
Applied 20241025030906/migrate users (56.077989ms)
error: while executing migration 20250409171833: error returned from database: CREATE INDEX CONCURRENTLY cannot run inside a transaction block
```

Reply:
thanks. I had only tested it by running the query directly. Lesson learned.